### DISCUSSION TITLE
How to get the SHA256 value?

### DISCUSSION THREAD

**Discussion by herdtm** [timbru31/mms-stockshock#1905] [Q&A] [answered]
From where can I get the SHA256 values?

From the browsers network traffic I can see API calls against the MMS API containing the SHA256. However, I am unable to find the values needed for the script:

loginSHA256 = ""
loginV2SHA256 = ""
categorySHA256 = ""
wishlistSHA256 = ""
addProductSHA256 = ""
searchSHA256 = ""

Help would be appreciated!

Wish you a happy new year!

**Comment by timbru31** [comment, marked as answer]
You need to take the SHA256 values from the MMS API calls you see in your browser's console. As I'm currently not running any stockshock instance I don't have the current values at hand, sorry.
